packages/api-database/source/models/transaction.ts

```diff
@@ -76,14 +76,14 @@ export class Transaction {
 		nullable: true,
 		type: "varchar",
 	})
-	public recipientId!: string | undefined;
+	public recipientId!: string | undefined | null;
```

Comment:
Can this be modified in a way to only return single type undefined or null in addition to the string?